### PR TITLE
terminal and menuitem all test cases

### PR DESCRIPTION
**Pull request by imAkshay26**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,88 +0,0 @@
-# PostgreSQL. Versions 9.1 and up are supported.
```
Comment (SakshiParakh) [inline config/database.yml]: Why is this file deleted?

Comment (imAkshay26) [inline config/database.yml]: i have modified my .gitignore file but still it is uploading on github so i deleted.Should i push once again??

Comment (imAkshay26) [inline config/database.yml]: It is present on my local git.

Comment (SakshiParakh) [inline config/database.yml]: I can't see it in the PR

Comment (SakshiParakh) [inline config/database.yml]: issue resolved ; used - git rm --cached filename

Comment (SakshiParakh) [inline config/database.yml]: issue resolved - used git rm --cached filename

**Comment by SakshiParakh** [comment]
The Factory girl syntax methods will be included in the next commit and accordinly changes will be made
